frame/staking/src/lib.rs

```diff
@@ -934,7 +949,7 @@ decl_storage! {
 			=> Option<StakingLedger<T::AccountId, BalanceOf<T>>>;
 
 		/// Where the reward payment should be made. Keyed by stash.
-		pub Payee get(fn payee): map hasher(twox_64_concat) T::AccountId => RewardDestination<T::AccountId>;
+		pub Payee get(fn payee): map hasher(twox_64_concat) T::AccountId => RewardPolicy<RewardDestination<T::AccountId>>;
```

Comment:
Just a note that this will need a migration script.

Reply:
yeah, unavoidable anyhow though. Either the name changes or the type or both.